third_party/move/move-compiler-v2/src/plan_builder.rs

```diff
@@ -316,7 +322,7 @@ fn parse_failure_attribute(
                         env.diag_with_labels(
                             Severity::Warning,
                             &attr_loc,
-                            BAD_ABORT_VALUE_WARNING,
+                            "Test will pass on an abort from *any* module.",
```

Comment:
Nice! I almost left a comment saying what was there before was a bit confusing, it took me a couple of reads to understand what was being conveyed. 

This is more readable and understandable IMO.